.github/workflows/build-docker.yaml

```
@@ -0,0 +1,43 @@
+name: Build and push spellcheck image
+
+on:
+  push:
+    branches:
+      - main
```

Let's trigger builds on PR just to check that the build works. (It is fast, so not much loss here)
```suggestion
  push:
    branches:
      - main
  pull-request:
    branches:
      - main
```